test/support/integration_test.rb

```diff
@@ -73,8 +78,9 @@ def clean_index_content(index)
   end
 
   def commit_document(index_name, attributes, id: attributes["link"], type: "edition")
-    insert_document(index_name, attributes, id: id, type: type)
-    commit_index(index_name)
+    insert_document(index_name, attributes, id: id, type: type).tap do
```

Comment:
Is `tap` being used to ensure that the return value is the ID returned by `insert_document`? If so, I think it would be much clearer to just assign a temporary variable - I had to read this a few times to work out why it was written like this (and I'm still not totally sure 🙂).

Reply:
Ya, not sure why I went with that approach.